exo-go/src/docker_compose/index.go

```diff
@@ -20,9 +20,9 @@ func PullAllImages(dockerComposeLocation string, log func(string)) error {
 	return processHelpers.RunAndLog(dockerComposeLocation, log, "docker-compose", "pull")
 }
 
-// RunAllImages runs all the docker images
-func RunAllImages(env []string, cwd string, log func(string)) (*processHelpers.Process, error) {
-	process := processHelpers.NewProcess("docker-compose", "up")
+// RunImages runs the given docker images
+func RunImages(images []string, env []string, cwd string, log func(string)) (*processHelpers.Process, error) {
+	process := processHelpers.NewProcess(append([]string{"docker-compose", "up"}, images...)...)
```

Comment:
I think this line can be simplified to `processHelpers.NewProcess("docker-compose", "up", images...)`